src/main/java/lib/math/differential/Derivative.java

```diff
@@ -24,6 +24,7 @@ public Derivative() {
 
     /**
      * Gets the derivative of the derivative.
+     *
      * @return The derivative of the derivative.
      */
     public Derivative differentiate() {
```

Comment:
Well technically yeah, but you can specify the time to the class instead of using FPGA timestamp, essentially giving it a series of points. But I'll write it anyway.

Reply:
My point is not the specific timestamp used, but that it's important to emphasize this is an integral over time—and not a general x. Anyway, looks good so resolving.